docs/plans/2026-02-24-fix-android-widget-background-refresh-plan.md

```
@@ -0,0 +1,151 @@
+---
+title: fix: refresh android widget after background lesson changes
+type: fix
+date: 2026-02-24
+---
+
+# fix: refresh android widget after background lesson changes
+
+## 🐛 Overview
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)